docs/components/plugins/api/APISectionUtils.tsx

```diff
@@ -306,6 +305,9 @@ export const getCommentOrSignatureComment = (
   signatures?: MethodSignatureData[]
 ) => comment || (signatures && signatures[0]?.comment);
 
+export const getTagData = (tagName: string, comment?: CommentData) =>
+  comment?.tags?.filter(tag => tag.tag === tagName)[0];
```

Comment:
This function seems useful only if CommentData is passed in -- make it the caller's responsibility not to call this function if the CommentData is falsy.

Reply:
I have tried refactor this method to the recommended approach, but it would result in type incompatibility, since everywhere else in types `CommentData` is marked as optional field, and passing any comment data produces error:
* https://github.com/expo/expo/blob/master/docs/components/plugins/api/APIDataTypes.ts

Also I don't think that marking `comment` as required and changing its type to `CommentData | undefined` is a good solution in this situation, so maybe we can leave the code as is?

Reply:
Ok. I still think it is better to lift this logic and write `comment ? getTagData(x, comment) : null` instead of having `getTagData` abstract this away.

Reply:
This might be a good idea, I will look at that and try to prepare separate PR with refactor in the future.